apps/ledger-live-mobile/src/components/CurrencyInput.tsx

```diff
@@ -241,7 +257,7 @@ const styles = StyleSheet.create({
   },
 });
 
-const mapStateToProps = state => ({
+const mapStateToProps = (state: Parameters<typeof localeSelector>[0]) => ({
```

Comment:
Would be more understandable to use `State` directly here

Reply:
Depends, if we ever change the type of localeSelector it will be inferred. So I'm not sure this is really relevant.